draft-ietf-dprive-dnsoquic.md

```diff
@@ -357,7 +358,7 @@ See {{iana-error-codes}} for details on registering new error codes.
 In QUIC, sending STOP_SENDING requests that a peer cease transmission on a
 stream. If a DoQ client wishes to cancel an outstanding request, it MUST issue
 a QUIC STOP_SENDING with error code DOQ_REQUEST_CANCELLED. This may be sent at
-any time but will be ignored if the server response has already been 
+any time but will have no effect if the server response has already been
 acknowledged. The corresponding DNS transaction MUST be abandoned.
 
```

Comment:
There is a related comment by @martinduke:
```
(5.3.1) Clients are allowed to send STOP_SENDING and servers are allowed to
send RESET_STREAM. Servers sending STOP_SENDING break the connection. Given the
prescriptiveness of these rules, it's odd that you don't address clients
sending RESET_STREAM. IMO this should be allowed, but either way it should be
specified.
```
There is another related comment by Robert Wilton:
```
I presume that there is no requirement that the DNS transaction be immediately
abandoned. E.g., if a server already has a reply queued for sending, then it
is still reasonable to send that response?
```
Again, need to find language that addresses these two other concerns.